jormungandr/src/explorer/graphql/mod.rs

```diff
@@ -63,16 +64,54 @@ impl Block {
     }
 
     /// The transactions contained in the block
-    pub fn transactions(&self, context: &Context) -> FieldResult<Vec<Transaction>> {
-        Ok(self
-            .get_explorer_block(&context.db)?
-            .transactions
-            .iter()
-            .map(|(id, _tx)| Transaction {
-                id: id.clone(),
-                in_block: self.hash.clone(),
-            })
-            .collect())
+    pub fn transactions(
+        &self,
+        first: Option<i32>,
+        last: Option<i32>,
```

Comment:
Is there meaning behind these parameters being potentially negative?

Reply:
Right; I thought these could be Pythonic wraparound indices, but this just looks like juniper being too Javascriptey.